nbproject/project.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be included in the project

Reply:
Solved